contracts/interchain-token-service/src/interface.rs

```diff
@@ -0,0 +1,58 @@
+use axelar_soroban_std::types::Token;
+use soroban_sdk::{contractclient, Address, Bytes, Env, String};
+
+use crate::error::ContractError;
+
+#[contractclient(name = "InterchainTokenServiceClient")]
+pub trait InterchainTokenServiceInterface {
```

Comment:
```suggestion
pub trait InterchainTokenServiceInterface: AxelarExecutableInterface {
```
extend the trait to enforce the methods

Reply:
updated